docs/backlog/backlog.rst

```diff
@@ -1,4 +1,52 @@
 Product Backlog
 ===========================================
+* Create App Icon/Name
+* Implement REST API for Communication with Artemis
+* Implement Authentication Functionality for Artemis Users
+  
+  - use Bearer Token / Cookie based Authentication based on Artemis Version
+* Implement a Functionality to view and switch between the Tutors Courses
+* Implement a View that shows all Exercises for a Course and allows selecting them
 
-by Alexander Lange 
\ No newline at end of file
+  - Show a Timeline indicating Release-, Submission- and Assesment (Due) Dates
+* Implement a View that shows Assessment Statistics for a specific Exercise
+* Implement a Functionality to search for a specific Students Submission
```

Comment:
"Implement a Functionality" sound weird IMO, I think it should just be "Implement functionality to..."